common/src/test/java/com/revenuecat/purchases/common/BackendTest.kt

```diff
@@ -1451,19 +1469,50 @@ class BackendTest {
         assertThat(headersSlot.captured["marketplace"]).isEqualTo("DE")
     }
 
+    @Test
+    fun `postReceipt sends pricingPhases`() {
+        val storeProduct = mockStoreProduct()
+
+        postReceipt(
+            responseCode = 200,
+            isRestore = false,
+            clientException = null,
+            resultBody = Responses.validFullPurchaserResponse,
+            observerMode = true,
+            receiptInfo = ReceiptInfo(
+                productIDs,
+                storeProduct = storeProduct
+            ),
+            storeAppUserID = null,
+            sendPricingPhases = true
+        )
+
+        assertThat(bodySlot.isCaptured).isTrue
+        assertThat(bodySlot.captured["pricing_phases"]).isEqualTo(listOf(mapOf(
+            "billingPeriod" to "P1M",
```

Comment:
could we pull the pricing phases to a var? we will probably reuse

Reply:
yes. i was waiting until we actually have a second use case to refactor